### ISSUE TITLE
Seedlink plotter kills my MacOs WindowServer !

### ISSUE THREAD

**Issue by sbonaime** [sbonaime/seedlink_plotter#39]
I just reinstalled Seedlink plotter with 
> pip install https://github.com/bonaime/seedlink_plotter/archive/master.zip

Then when I try this command
seedlink-plotter --seedlink_server "rtserver.ipgp.fr:18000" -s "G_SSB:10HHZ" --line_plot -b 50  -v
I have a black screen and then the MacOs Login interface. seedlink-plotter just killed my WindowServer !

MacOs Mojave10.14.6

````
INFO: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:connected to: 'SeedLink v3.2 (2014.071)'
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:sending: STATION  SSB G
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:response: station is OK (selected)
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:sending: b'SELECT 10HHZ'
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:response: selector b'10HHZ' is OK
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:response: 1 selector(s) accepted
INFO: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:requesting specified time window
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:sending: b'TIME 2019,9,3,15,49,39'
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:response: DATA/FETCH/TIME command is OK
INFO: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:1 station(s) accepted
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:sending: END
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:primary loop pass 0, 2019-09-03 19:50:46.911 python[8014:401917] HIToolbox: received notification of WindowServer event port death.
2019-09-03 19:50:48.666 python[8014:401917] port matched the WindowServer port created in BindCGSToRunLoop
state 2
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:primary loop pass 0, state 2
INFO:root:0 Trace(s) in Stream:

ERROR:root:Empty stream for plotting
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:primary loop pass 0, state 2
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:primary loop pass 0, state 2
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:primary loop pass 0, state 2
DEBUG: obspy.clients.seedlink [rtserver.ipgp.fr:18000]:primary loop pass 0, state 2
```

**Comment by sbonaime** [comment]
Tkinter app on macOS 10.14.6 problem
https://github.com/pyinstaller/pyinstaller/issues/4334

**Comment by megies** [comment]
Ouuucch

**Comment by sbonaime** [comment]
Temporary fix is to downgrade python to 3.7.0

**Comment by sbonaime** [comment]
Fixed with Catalina 10.15.1
